README.md

```diff
@@ -68,18 +70,17 @@ right), and download the package.
 Snapshots are dynamically-linked x86\_64 builds, you'll need additional
 packages installed via your package manager.
 
-#### Fedora
-
-    $ sudo dnf install libpng SDL2 SDL2_net opusfile
+### Fedora
```

Comment:
I think I made an error earlier - sections Linux, Windows, macOS should be subsections of "Development snapshot builds", and this would keep sections per distro at level 4, otherwise it might be visually hard to distinguish where next section starts.

Change it to:
```
### [Linux](…)

…

#### Fedora

…

### [Windows](…)

…
```
This way linter probably won't complain about nesting structure.

Reply:
Will do!

Reply:
Yes, nice and consistent now.